tests/benchmarks/micro_benchmarks/test_cuda_memory_bw_performance.py

```diff
@@ -46,8 +46,7 @@ def test_cuda_memory_bw_performance(self):
 
```

Comment:
I don't think we need to change test because we just revise the structure. The test should be the same.

Reply:
sorry, restored